__tests__/app.test.js

```diff
@@ -73,3 +74,35 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("200: with an array of aricleswith the correct properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
```

Comment:
You need to check the length of the array first. If you use a forEach on an empty array, it will not run so your assertions will be skipped over.